### PR TITLE
📝 Added new skills and their icon links 🔗

### PR DESCRIPTION
**Pull request by Maddoxx88**
Reply if someone has new skills and their icon links (SVG Only)

### CODE REVIEW

File: src/constants/skills.js [hunk 1]
```diff
@@ -127,11 +128,21 @@ const icons = {
     blender: 'https://download.blender.org/branding/community/blender_community_badge_white.svg',
     babel: 'https://www.vectorlogo.zone/logos/babeljs/babeljs-icon.svg',
     sketch: 'https://www.vectorlogo.zone/logos/sketchapp/sketchapp-icon.svg',
+    flask: 'https://www.vectorlogo.zone/logos/pocoo_flask/pocoo_flask-ar21.svg',
```
Comment (rahuldkjain) [inline src/constants/skills.js:131]: Please use this instead: https://www.vectorlogo.zone/logos/pocoo_flask/pocoo_flask-icon.svg
Do not use icons with text.

File: src/constants/skills.js [hunk 2]
```diff
@@ -127,11 +128,21 @@ const icons = {
...
+    nativescript: 'https://raw.githubusercontent.com/detain/svg-logos/780f25886640cef088af994181646db2f6b1a3f8/svg/nativescript.svg',
+    xamarin: 'https://raw.githubusercontent.com/detain/svg-logos/780f25886640cef088af994181646db2f6b1a3f8/svg/xamarin.svg',
+    vagrant: 'https://www.vectorlogo.zone/logos/vagrantup/vagrantup-ar21.svg',
+    tailwind: 'https://www.vectorlogo.zone/logos/tailwindcss/tailwindcss-ar21.svg',
```
Comment (rahuldkjain) [inline src/constants/skills.js:135]: use this https://www.vectorlogo.zone/logos/tailwindcss/tailwindcss-icon.svg

File: src/constants/skills.js [hunk 3]
```diff
@@ -127,11 +128,21 @@ const icons = {
...
+    materialize: 'https://raw.githubusercontent.com/prplx/svg-logos/5585531d45d294869c4eaab4d7cf2e9c167710a9/svg/materialize.svg',
+    invision: 'https://www.vectorlogo.zone/logos/invisionapp/invisionapp-ar21.svg',
```
Comment (rahuldkjain) [inline src/constants/skills.js:137]: use this: https://www.vectorlogo.zone/logos/invisionapp/invisionapp-icon.svg

File: src/constants/skills.js [hunk 4]
```diff
@@ -127,11 +128,21 @@ const icons = {
...
+    framer: 'https://www.vectorlogo.zone/logos/framer/framer-ar21.svg',
```
Comment (rahuldkjain) [inline src/constants/skills.js:138]: use this: https://www.vectorlogo.zone/logos/framer/framer-icon.svg

**Comment by Maddoxx88** [comment]
Done @rahuldkjain

**Pull request merged**